device_services/generic/chargepoint.md

```diff
@@ -27,6 +27,10 @@ Chargepoint service is used to represent EV chargers.
 | in   | cmd.max_current.set             | int        |                                                                     | Sets the maximum offered current in `A`, must be an integer between `6` and `sup_max_current` service property value. |
 | in   | cmd.max_current.get_report      | null       |                                                                     | Requests the maximum offered energy.                                                                                  |
 | out  | evt.max_current.report          | int        |                                                                     | Reports the maximum offered current in `A`, this is effectively a static load balancing value.                        |
+| -    |                                 |            |                                                                     |                                                                                                                       |
```

Comment:
'max_current' is referred but never define in this service

Reply:
What? It's not part of this PR.